### PR TITLE
opencv properties and coco.names

### PR DESCRIPTION
**Pull request by supertick**

### CODE REVIEW

File: src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java [hunk 1]
```diff
@@ -90,8 +79,8 @@ public OpenCVFilterYolo() {
     this(null);
   }
 
-  private void downloadYoloModel() {
-    log.info("downloadYoloModel - begin");
+  private void checkFiles() {
+    log.info("checkFiles - begin");
     File yoloHome = new File(darknetHome);
     if (!yoloHome.exists()) {
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java:85]: this method isn't downloading files anymore.. it shouldn't create the files.  additionally, this method doesn't return and value, so the caller of this method doesn't know if the files were valid.  This method should either be removed, or it should include a return value that has some useful meaning.

File: src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java [hunk 2]
```diff
@@ -105,66 +94,23 @@ private void downloadYoloModel() {
...
-    }
-    log.info("downloadAndCache - end");
+    log.info("checkFiles - end");
   }
 
   private void loadYolo() {
     log.info("loadYolo - begin");
 
     // If the model isn't there, we should download it and cache it.
     log.info("Staritng yolo download verification");
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java:112]: this log message is wrong, it doesn't download anymore..  same with the following log message.  and there's no validation that "checkFiles" was successful.

File: src/main/java/org/myrobotlab/service/OpenCV.java [hunk 3]
```diff
@@ -455,7 +455,7 @@ static public ServiceType getMetaData() {
     meta.addDependency("com.github.axet", "vget", "1.1.34");
 
     // yolo models
-    meta.addDependency("yolo", "yolov2", "v2", "zip");
+    meta.addDependency("yolo", "yolov2", "v3", "zip");
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/OpenCV.java:458]: When did we start supporting yolo v3?!  Was that added recently?  I didn't think that was supported in the current version of opencv...

File: src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java (hunk 1 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java:85]: will remove it

File: src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java (hunk 2 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/opencv/OpenCVFilterYolo.java:112]: this is dead to me...

File: src/main/java/org/myrobotlab/service/OpenCV.java (hunk 3 above)
Comment (supertick) [reply src/main/java/org/myrobotlab/service/OpenCV.java:458]: changing to "yolo", "yolov2", "0.0.2"

**Pull request merged**